projects/dev-schematics/README.md

```diff
@@ -62,10 +62,10 @@ For ease of usage we provide scripts for smoother installation process:
 There might be situatons, where you want to add new entities to/from public API of schematics, but the @spartacus/schematics hasn't been released yet. You can test it locally by following steps:
 
 1. Launch local verdaccio in yours terminal with command `verdaccio` 
-2. Second step has 2 possible ways:
-    - from project root go to scripts/ and run `./publish-schematics-verdaccio.sh` script, which will automatically do all the work for you with building and publishing your changes to local verdaccio
-    - go to `/project/schematics` and run `yarn build && npm publish --registry http://localhost:4873`
-3. Go to dev-schematics project folder and create new file `.npmrc` with content: `@spartacus:registry=http://localhost:4873`
-4. With verdaccio launched, local schematics published to it, we can install new version for testing: `rm -rf node_modules && yarn`
-    
+2. From project root go to `/projects/schematics` and change lib version in package.json file (for local testing purposes only, please remember to revert it before committing any changes)
```

Comment:
Is there a restriction to which version should we change it? E.g. should the version be greater than the current one?

Reply:
Well, I believe at first thought I would say Verdaccio works only with bumped version, but on the second one - we're publishing the previous one, so verdaccio isn't aware of other versions to be able to compare with, right?

Reply:
I think the version on the verdaccio should be greater than the version in the testing project.
And it doesn't really matter which version is it, as long as it's greater. If you can verify this, we can then simplify the process and make [this one](https://github.com/SAP/spartacus/pull/7554#discussion_r431727063) optional.

Reply:
Okay, checked. There is no need to provide modified version for lib, as we're republishing the whole lib.